server/chat-plugins/scavengers.ts

```diff
@@ -1853,12 +2159,16 @@ const commands = {
 			return this.privateModAction(`Hint #${hintNumber} was removed from Recycled hunt #${huntNumber} question #${questionNumber}.`);
 		} else if (cmd === 'autostart') {
 			if (params[0] !== 'on' && params[0] !== 'off') return this.errorReply(usageMessages[cmd]);
-			if (params[0] === 'on' && room.addRecycledHuntsToQueueAutomatically || params[0] === 'off' && !room.addRecycledHuntsToQueueAutomatically) return this.errorReply(`Autostarting recycled hunts is already ${room.addRecycledHuntsToQueueAutomatically ? 'on' : 'off'}.`);
-			room.addRecycledHuntsToQueueAutomatically = !room.addRecycledHuntsToQueueAutomatically;
+			if (params[0] === 'on' && room.scavSettings.addRecycledHuntsToQueueAutomatically || params[0] === 'off' &&
+				!room.scavSettings.addRecycledHuntsToQueueAutomatically
+			) {
```

Comment:
```suggestion
			if ((params[0] === 'on') === !!room.scavSettings.addRecycledHuntsToQueueAutomatically) {
```

Reply:
This is my secret trick for "concisely see if two settings match" - just see if they're equal to each other. :3